playground/android/app/src/main/res/values/styles.xml

```diff
@@ -3,6 +3,7 @@
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <!-- Customize your theme here. -->
+        <item name="android:textColor">#000000</item>
```

Comment:
is this on purpose?

Reply:
On purpose, yes, as part of the official RN migration guide.
Adds real value: meh.

Reply:
try to see if the app looks decent without it and if so, better remove... better not custom-theme if no need to